.github/workflows/docker-promote.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: docker promote
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_tag:
+        description: 'Branch version number'
+        required: true
+        default: '1.8'
```

Comment:
I don't think we should have defaults for these values. Whoever uses these scripts should take care to choose the right version.

Reply:
Fixed